### ISSUE TITLE
TypeError: list indices must be integers or slices, not float

### ISSUE THREAD

**Issue by bejucoo** [peterkuma/ccplot#9]
Hello,

when I use the -x option plotting a CALIPSO profile with the command

`ccplot -x +0:00..+1:00 -o calipso532.png -c calipso-backscatter.cmap calipso532 CAL_LID_L1-Standard-V4-10.2020-02-01T06-25-30ZN_Subset.hdf`

I get the following error:

```
Traceback (most recent call last):
  File "/usr/local/bin/ccplot", line 2321, in <module>
    main(sys.argv)
  File "/usr/local/bin/ccplot", line 2196, in main
    opts = parse_options(argv)
  File "/usr/local/bin/ccplot", line 910, in parse_options
    opts.hextent = parse_extent(a)
  File "/usr/local/bin/ccplot", line 808, in parse_extent
    e.relative_time[k/4] = sign*dt.timedelta(0, seconds,
TypeError: list indices must be integers or slices, not float
```
I'm not sure how to fix it. If I change the values I get a `ccplot: Invalid argument passed to -x:` error. Maybe it has something to do with [floor division](https://stackoverflow.com/questions/21609227/python-3-3-3-typeerror-list-indices-must-be-integers-not-float)? I'm using Ubuntu 20.1 and

```
ccplot 1.5.5

Third-party libraries:
matplotlib 3.3.4
basemap 1.2.2+dev
```

**Comment by peterkuma** [comment]
Thank you for reporting this issue. It was related to integer division as you say, which is different between Python 2 and Python 3. It should be fixed in the master branch of the repository, and I will make a new release soon.

**Comment by peterkuma** [comment]
It should be fixed in ccplot 1.5.6.
